src/sats_management.c

```diff
@@ -129,9 +130,11 @@ static void set_reference_sat(const u8 ref_prn, sats_management_t *sats_manageme
     memcpy(old_prns, sats_management->prns, sats_management->num_sats * sizeof(u8));
```

Comment:
Could you add a few comments to this function for future explorers? I feel like this may not be the last time this function gets revisited...